dot/build_spec_integration_test.go

```diff
@@ -0,0 +1,165 @@
+//go:build integration
+// +build integration
+
+// Copyright 2020 ChainSafe Systems (ON) Corp.
```

Comment:
nit although we'll probably use #1997 soon

```suggestion
// Copyright 2021 ChainSafe Systems (ON) Corp.
```